libs/designer-ui/src/lib/workflowparameters/workflowparameters.less

```diff
@@ -79,6 +79,7 @@
   }
 
   .msla-workflow-parameters-link {
+    position: absolute;
```

Comment:
Will this overlap the wfp content if it's long enough?  We might either need to have a footer component with a solid background to place this in

Reply:
I can check that scenario. Thanks!